text-ui-test/input.txt

```diff
@@ -0,0 +1,14 @@
+todo read book
+list
+deadline return book /by Sunday
+event project meeting /at Mon 2-4pm
+list
+mark 2
+mark 3
+delete 3
+list
+unmark 3
+mark 1
+list
+bye
+
```

Comment:
You could consider adding some incorrect commands and see how Duke responds to them